api/services/buildEngines.js

```diff
@@ -84,7 +131,8 @@ module.exports = {
           branchURL: defaultBranch ? '' : '/' + model.branch,
           root: defaultBranch ? 'site' : 'preview'
         },
-        template = _.template(cmd.join(' && '));
+        // Temporary until workaround for single line IF EXIST logic is implemented
```

Comment:
Sure thing. So currently the code concatenates each of the run commands in the array into a single line with each command separated by an `&&`. On Windows, the `RMDIR` command cannot check the existence of a directory before it exists so if it tries to delete a non-existing directory, it throws an error message. If you concatenate the commands with `&` on Windows, it will run each command regardless of whether or not the previous command completes successfully which is what `&&` imposes.

To prevent `RMDIR` from failing on non-existent directories, one could execute an `IF EXIST <directory> RMDIR <directory>' command. However, Windows does *not* support concatenating this command with other commands on the same line as follows:

```
IF EXIST <dir> RMDIR <dir> && MKDIR <newdir>
```